automatic/winscp.install/update.ps1

```diff
@@ -4,11 +4,22 @@ $releases = 'https://winscp.net/eng/downloads.php'
 $re  = 'WinSCP.+\.exe/download$'
 
 function global:au_SearchReplace {
+    $FileVersion = if ($Latest.FileVersion -ne $Latest.Version) {
+        $Latest.FileVersion
+    } else {
+        '$($env:ChocolateyPackageVersion)'
+    }
+
    @{
         "$($Latest.PackageName).nuspec" = @{
             "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`$2"
         }
-
+        "tools\chocolateyInstall.ps1"   = @{
+            # We use ChocolateyPackageVersion to minimise changes to the package code -
+            # If using a patchfix version, use the fileversion instead of the packageversion.
+            '(?i)(\s*file\s*=\s*["'']\$toolsPath\\WinSCP-)(\$\(\$env:ChocolateyPackageVersion\)|.+)(-Setup\.exe["''])' = "`${1}$($FileVersion)`${3}"
+            '(?i)(\s*file64\s*=\s*["'']\$toolsPath\\WinSCP-)(\$\(\$env:ChocolateyPackageVersion\)|.+)(-Setup\.exe["''])' = "`${1}$($FileVersion)`${3}"
+        }
```

Comment:
No need to use use such a cumbersom way to set the file name.
Since we use `Get-RemoteFiles` it also sets the variables `FileName` and `FileName64` that can be used here.

```suggestion
        "tools\chocolateyInstall.ps1"   = @{
            "(?i)(^\s*file\s*=\s*`"[$]toolsPath\\).*"   = "`${1}$($Latest.FileName)`""
            "(?i)(^\s*file64\s*=\s*`"[$]toolsPath\\).*" = "`${1}$($Latest.FileName64)`""
        }
```

Reply:
On reflection, I'm unsure exactly why we have both file and file64 being set.